### ISSUE TITLE
Escape quotes for PRs titles

### ISSUE THREAD

**Issue by ahmed2m** [eamodio/vscode-gitlens#1308]
- GitLens Version: v2021.1.504
- Git Version: 2.29.2
- VSCode Version: 1.52.1
- OS Version: Linux Manjaro Kernel 5.10.2-2

I've used a tiny reproduction repo for this issue https://github.com/ahmed2m/gitlens-issue

Steps to Reproduce:

1. Clone the repo above.
2. open folder-1 > readme.md
3. hover to get the gitlens tooltip on any part of any line.
4. The link part that supposed to say `via PR #1` is messed up because of unescaped string (the PR title has double quotes in it)
![image](https://user-images.githubusercontent.com/26524089/103712661-1ab1a980-4fc3-11eb-81be-e3fedc82e23f.png)

Here's the text from the screenshot:
```
via [PR #1](command:gitlens.action.openPullRequest?%7B%22pullRequest%22%3A%7B%22id%22%3A%221%22%2C%22provider%22%3A%22GitHub%22%2C%22repoPath%22%3A%22%2Fhome%2Fahmed%2Fplayground%2Fgitlens-issue%22%2C%22url%22%3A%22https%3A%2F%2Fgithub.com%2Fahmed2m%2Fgitlens-issue%2Fpull%2F1%22%7D%2C%22type%22%3A%22openPullRequest%22%7D "Open Pull Request #1 on GitHub —— very "important" feature Merged, 20 minutes ago")   (January 6th, 2021 1:40am)
```

**Comment by eamodio** [comment]
Can you please verify this fix in tomorrow's insiders edition?

You can install the insiders edition from [here](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens-insiders). Be sure to disable/uninstall the stable version of GitLens first.

**Comment by ahmed2m** [comment]
@eamodio It's fixed for me now on `v2021.1.604`
Thank you.
